demo/src/store/store.js

```diff
@@ -41,10 +41,43 @@ function reducer(state = {}, action) {
   return newState;
 }
 
-export const store =
-    createStore(
-        reducer,
-        initialState,
-        window.__REDUX_DEVTOOLS_EXTENSION__ &&
-            window.__REDUX_DEVTOOLS_EXTENSION__(),
-    );
+/**
+ * Save a state to localStorage.
+ * @param {{items:ItemStore}} store The new state.
+ */
+const saveState = (store) => {
+  try {
+    const updatedStore = JSON.stringify(store);
+    localStorage.setItem('state', updatedStore);
+  } catch (err) {
+    // for some reason we can't save, just log an error message
+    console.error('There was an error saving the state:\n' + err);
```

Comment:
(assuming it's based on Error, which it should be). This will stringify the error to just the err.message, and log a stack trace to this console statement.

However, if you log the error on its own in a second statement, e.g. console.error(err), that will log the stack trace to the error as well, indicating which line the error originally happened on.

Example console outputs:
![image](https://user-images.githubusercontent.com/30128133/84949838-b7057480-b0a2-11ea-9bbc-e8bf4c732016.png)

Reply:
Interesting, fixed